README.md

```diff
@@ -32,6 +32,38 @@ First fork the [DjangoGirls/tutorial](https://github.com/DjangoGirls/tutorial) r
 
 ![Fork button](contributing/images/fork.png)
 
+## CLI for Development
+
+This command line tool use `make` to create development environment. It is optional to use this tool. While building the document, it builds the document for every language. The build process can be limited to any one language using this tool and reduce build time considerbly. Afterwards, the translation to other languages are done from crowdin localization process.
+
+```
+Usage: make command [LANG=<language_short_code>]
+
+    LANG: Language shortcodes are found in LANGS.md. Default is 'en'(English).
+           ( Refer LANGS.md for shortcodes of Languages available. )
+
+Commands:
+  help      - Display make command list.
+  dev       - Setup the project and start the development server with debugging enabled.
+  check     - Check for the root directory for various dependencies.
+  setup     - Setup the temporary language and install node dependencies for the development.
+  build     - Build the honkit project.
+  build-dev - Build the honkit project with debug log.
+  serve     - Start honkit server locally for development.
+  pdf       - Generate the PDF version of DjangoGirls tutorial.
+  epub      - Generate the EPUB version of DjangoGirls tutorial.
+  mobi      - Generate the MOBI version of DjangoGirls tutorial.
+  mode      - Shows the development mode status.
+  exit      - Exit development mode.
+
+Example:
+
+$ make dev LANG=es
+
+The above command will start the development server using the language Español.
+'LANG' argument is only required the first time until the exit command is executed.
+```
+
```

Comment:
Is this required to be mentioned in README?

Reply:
*Some* mention in the README is probably good to have. Though if we mention the whole usage instructions, we'll also have to keep them in sync with future evolution of the Makefile.

Maybe it would suffice to just mention that usage instructions are available though `make help`?

Reply:
Sounds good. Will make changes and push the final commit.

Reply:
Resolved with [e8742d9](https://github.com/DjangoGirls/tutorial/pull/1729/commits/e8742d979c92497d929d011822b8925d12f85dd2)